docs/mcu.md

```diff
@@ -322,7 +322,7 @@ For example, some device might be able to send the line to the display asynchron
 DMA. In that case, we'd have two line buffer. One working line, and one which is being send
 to the screen, in parallel.
 
-## Our supported boards
+## Supported Boards
```

Comment:
I I think we should not use that title, we kind of want to emphasis that all board are supported, maybe something like `Tested Boards`  or `Example Boards` ?

Reply:
I used `Example Implementations`